student.rb

```diff
@@ -0,0 +1,12 @@
+require 'person'
+
+class Student < Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- [ ] [Optional] If you applied the recommendation I mentioned above about the `id`, you don't need it here so please delete the `id` from lines 4 and 5 here
- [ ] [Required] Please add `classroom` to the `initialize` method parameter as required [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=from%20Person.-,Constructor%20extends%20parent%27s%20constructor%20by%20adding%20%40classroom%20and%20a%20parameter%20for%20it.,-Method%20play_hooky%20that).

[Hint] It will be like that:
```ruby
 def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
 end
```